Makefile

```diff
@@ -156,6 +156,18 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/crd | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
+.PHONY: install-cert-manager
+install-cert-manager:
+    @cmctl check api 2>&1 | grep -q 'The cert-manager API is ready' && echo "Cert Manager is already installed" || \
+        (echo "Cert Manager is not installed. Installing Cert Manager..." && \
+        kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml)
```

Comment:
Good point. I think the YAML at given URL (containing version) still should be available for the backward compatibility purposes. 

I can specify the version inside a constant but I would still prefer to retain an explicit version (for which we are qualifying the NDB operator). Any upgrades in cert-manager versions should be explicit. 

If the version goes E0L, we will get an explicit error and then we can upgrade.

Reply:
Decided to remove the target itself as running the target is not sufficient. We need to wait until all the pods are up inside k8s cluster before running the next "make deploy" command. So I will add a note to mention the pre-requisite steps in README, instead of relying on this target.